CONTRIBUTING.md

```diff
@@ -1,17 +1,65 @@
-Contributing
-============
+# Contributing
 
-TODO: Fill out your contributing process, or just stick with the default.
-
-Steps
------
+## Steps
 
 Thanks for your interest in sensu-go cookbook! If you'd like to help out please:
 
 1. Fork it
 1. Create your feature branch (`git checkout -b my-new-feature`)
-1. Add tests for the new feature; ensure they pass (`rake`)
+1. Add tests for the new feature; ensure they pass See [TESTING.md](./TESTING.md)
 1. Add a changelog entry under `## [Unreleased]` and the appropriate sub header for the type of change per our [guidelines](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
 1. Commit your changes (`git commit -am 'Add some feature'`)
 1. Push to the branch (`git push origin my-new-feature`)
 1. Create a new Pull Request
+
+## How to Become a Maintainer
+
+### Types of Maintainers
```

Comment:
Not sure if it makes sense to go into the nuances that I don't think apply here.